listen.py

```diff
@@ -43,6 +43,41 @@ def from_json(cls, j):
             data=j.get('track_metadata')
         )
 
+    @classmethod
+    def from_influx(cls, row):
+        """ Factory to make Listen objects from an influx row
+        """
+        dt = datetime.strptime(row['time'] , '%Y-%m-%dT%H:%M:%SZ')
+        t = int(dt.strftime('%s'))
+        mbids = []
+        if row.get('artist_mbids'):
+            for mbid in row.get('artist_mbids').split(','):
+                mbids.append(mbid)
```

Comment:
Fixed, I've named the variables `artist_mbids` and `influx_tags`. Is that okay, or should I use some other names?